src/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -59,7 +59,7 @@ static_library("SiWx917") {
     "../../FreeRTOS/SystemTimeSupport.cpp",
     "../../SingletonConfigurationManager.cpp",
     "../rs911x/BLEManagerImpl.cpp",
-    "../rs911x/rsi_ble_config.h",
+    "../rs911x/ble_config.h",
```

Comment:
As stated on PR #28752 ble_config.h should be removed completely and the necessary `#ifdef` should be added instead

Reply:
added #ifdef